### ISSUE TITLE
Reporting content not showing in various places

### ISSUE THREAD

**Issue by xaqrox** [radcampaign/pulitzer2016#149]
A dummy story, Test Story: Last Hospital in Aleppo Falls to Assad (http://dev.pulitzercenter.org/reporting/test-story-last-hospital-aleppo-falls-assad) had some issues in how it appears throughout the site.

It **does** show up on the On War and Peace Gateway landing page correctly: (http://dev.pulitzercenter.org/gateway/on-war-and-peace)

It also shows up correctly on the American University Campus Consortium partner page and Campus Consortium partner landing page. 

However, it **does not** show up correctly on the Reporting landing page or its filtered subpages:
* Reporting landing page: http://dev.pulitzercenter.org/reporting
* Articles landing page: http://dev.pulitzercenter.org/reporting?f[0]=field_vocab_media%3A1484
* Middle East landing page: http://dev.pulitzercenter.org/reporting?f[0]=field_vocab_media%3A1484&f[1]=taxonomy_vocabulary_3%3A8
* 2016 landing page: http://dev.pulitzercenter.org/reporting?f[0]=field_vocab_media%3A1484&f[1]=taxonomy_vocabulary_3%3A8&f[2]=created%3A2016
* New Yorker stories landing page page: http://dev.pulitzercenter.org/reporting?f[0]=field_vocab_media%3A1484&f[1]=taxonomy_vocabulary_3%3A8&f[2]=created%3A2016&f[3]=taxonomy_vocabulary_13%3A1532
* Syria country landing page: http://dev.pulitzercenter.org/reporting?f[0]=field_vocab_media%3A1484&f[1]=taxonomy_vocabulary_3%3A8&f[2]=created%3A2016&f[3]=taxonomy_vocabulary_13%3A1532&f[4]=taxonomy_vocabulary_12%3A875

We tried an additional test story, http://dev.pulitzercenter.org/reporting/test-story-something-about-fractured-lands, and it had the same issues.

**Comment by xaqrox** [comment]
I figured this out. But it doesn't make sense. The Reporting view uses the search api node index as its base table. New nodes get queued to be indexed when they are created. The indexing happens in batches on each cron.

In a happy coinicidence, I *just* learned about how cron and queues work together in the course of thinking about how to deal with #165. Also in that process I noticed that cron seemed to be failing because of the Lesson Builder redirect (#164).

My theory now is that nodes were not getting indexed because every time cron would have indexed them, it got redirected to the lesson builder. But if that's the case, I don't know how anything ever ended up getting indexed. In other words, I'm not sure this is a regression. But it's definitely a problem. Anyway, I've committed a fix for #164; we'll see if that lets cron run appropriately now.
